### PR TITLE
Changed example to fit the solution

### LINKED ISSUE #50450: SASS Nesting
### Describe the Issue

As per the question user has to organize the css rule to  nest h1 and p elements in .blog-post . But in solution if user does not nest h1 then it showing correct answer

### Affected Page

https://www.freecodecamp.org/learn/front-end-development-libraries/sass/nest-css-with-sass
https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/03-front-end-development-libraries/sass/nest-css-with-sass.md

### Your code

```
.blog-post {
    h1{
      text-align: center;
      color: blue;
    
    p{
      font-size: 20px;
    }
  }
  }


```

### Expected behavior

This code should run correctly as per question

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by pushkar707**
Checklist:

- [ ] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [ ] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #50450

### CODE REVIEW

File: curriculum/challenges/english/03-front-end-development-libraries/sass/nest-css-with-sass.md [hunk 1]
```diff
@@ -13,31 +13,31 @@ Sass allows nesting of CSS rules, which is a useful way of organizing a style sh
...
+  p {
+    color: white;
+  }
 
-    li {
-      display: inline-block;
-    }
+  ul {
+      color: blue;
   }
```
Comment (lasjorg) [inline curriculum/challenges/english/03-front-end-development-libraries/sass/nest-css-with-sass.md:41]:
```suggestion
  ul {
    color: blue;
  }
```

Comment (lasjorg) [reply curriculum/challenges/english/03-front-end-development-libraries/sass/nest-css-with-sass.md:41]: Please remove the extra spaces.

**Review by lasjorg — CHANGES_REQUESTED** [review]

**Comment by jeremylt** [comment]
This wasn't the example given in the linked issue.

Please use the changes agreed upon in the issue.

Also, please read and follow the [contribution guidelines](https://contribute.freecodecamp.org/) and the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).

**Comment by lasjorg** [comment]
I would be fine with using @pushkar707 example. I would prefer if the div was a `section` or `article` element but that isn't super important.

I think using the descendant combinator is probably a good idea anyway.


```
article {
  height: 200px;
}

article p {
  color: white;
}

article ul {
  color: blue;
}
```

Or (I'm not married to the actual styles, they are just an example)

```
.products {
  width: min(1400px, 90%);
  margin-inline: auto;
}

.products article {
  padding: 1rem;
}

.products p {
  font-size: 1.2rem;
  line-height: 1.4;
}
```

I'm fine with either.

**Comment by pushkar707** [comment]
> I would be fine with using the example from @pushkar707. I would prefer if the div was a `section` or `article` element but that isn't super important.
> 
> I think using the descendant combinator is probably a good idea anyway.
> 
> 
> ```
> article {
>   height: 200px;
> }
> 
> article p {
>   color: white;
> }
> 
> article ul {
>   color: blue;
> }
> ```
> 
> Or (I'm not married to the actual styles, they are just an example)
> 
> ```
> .products {
>   width: min(1400px, 90%);
>   margin-inline: auto;
> }
> 
> .products article {
>   padding: 1rem;
> }
> 
> .products p {
>   font-size: 1.2rem;
>   line-height: 1.4;
> }
> ```
> 
> I'm fine with either.
> 
> ---
> 
> > I fixed the linking to the issue.
> 
> Thanks for catching that.

Okay @lasjorg,  I will fix the spaces and use your example. Should I also make a new Pr from a seperate branch or should I update this one?

**Comment by jeremylt** [comment]
> please read and follow the contribution guidelines and the how to open a pull request guide.

These documents will tell you how to name your PR ;)

I keep unchecking boxes for you because if your PR isn't named per our naming conventions, then you haven't read these documents!

**Pull request closed without merging**